website/docs/using-unleash/troubleshooting/flag-abn-test-unexpected-result.md

```diff
@@ -0,0 +1,33 @@
+---
+title: My feature flag is returning unexpected results in my A/B/n test
+---
+
+Occasionally, users might come across a situation where a feature flag is returning unexpected results when running an A/B/n test.
+
+The first thing to look into is your feature configuration.
+
+If you're unsure whether the feature flag is being properly returned, you can go through the steps in this troubleshooting guide before proceeding: [My feature flag is not returned in the Frontend API/Edge/Proxy](/using-unleash/troubleshooting/flag-not-returned.md).
+
+1. If you're using a [gradual rollout activation strategy](/reference/activation-strategies#gradual-rollout) be sure to set the rollout percentage to the percentage of your user base you would like to target for the test. If you would like to target 100% of your user base on your A/B/n test, confirm that your rollout percentage is set to 100%. This adjustment guarantees that all users participate in the A/B/n test, irrespective of the division ratio, be it an even 50-50 split, for a typical A/B test with 50% for Group A and 50% for Group B, or any other distribution that you configure in your variants.
```

Comment:
Up to you if you want to include this but part of my debugging strategy is often to throw away gradual rollout and use the standard strategy. Can't be off when your logic is `true === true`;

Other things that might be useful to point out - know what your stickiness is. The default, you can use `userId` or `sessionId` but one of those _must_ be on your context to get consistent resolutions.

Last thing (phew), if you use custom stickiness and don't pass that property, your toggle will always be off (and you'll get the default variant)

Reply:
Awesome! Love the change!